recipes/poco/all/conanfile.py

```diff
@@ -29,12 +29,14 @@ class PocoConan(ConanFile):
         "fPIC": [True, False],
         "enable_fork": [True, False],
         "enable_active_record": [True, False, "deprecated"],
+        "log_debug": [True, False],
```

Comment:
The `log_debug` option could be deleted in the `config_options` method when building Debug and read with `get_save` below. But I think creating the package will be more challenging then...